proposals/incubation/openfeature.md

```diff
@@ -30,10 +30,35 @@ The community statistics are extracted from [CNCF devstats](https://openfeature.
 ### Technical
 
 #### SDK Progress
+Since achieveing sandbox we have progressed in the development of our SDKs:
+- [.NET](https://github.com/open-feature/dotnet-sdk) released v1.1.0
+- [Java](https://github.com/open-feature/java-sdk) released v1.2.0
+- [PHP](https://github.com/open-feature/php-sdk) released v2.0.0
+- [Python](https://github.com/open-feature/python-sdk) released v0.0.9
```

Comment:
Both Python and Ruby are pre 1.0. Do you think we should put them on a different list?

Reply:
not sure, it is the general advancement of the project. Happy to hear suggestions